hck_aux.sh

```diff
@@ -359,3 +359,12 @@ kill_ivshmem_server() {
     sudo kill ${cat ${IVSHMEM_PID}}
   fi
 }
+
+disable_bridge_nf() {
+  if [ "$DISABLE_BRIDGE_NF" = "on" ]
+  then
+    sysctl net.bridge.bridge-nf-call-arptables=0 > /dev/null
```

Comment:
Print should be here

Reply:
I was following the style that was already used in code:

hck.sh:
```
  echo Disabling bridge-netfilter...
  disable_bridge_nf

  echo Creating bridges...
  create_bridges
```

hck_aux.sh:
```
create_bridges() {
  case $TEST_NET_TYPE in
  bridge)
     brctl addbr ${TEST_BR_NAME} 2>&1 | grep -v "already exists"
     ifconfig ${TEST_BR_NAME} up
     ifconfig ${TEST_BR_NAME} txqueuelen $(queue_len_tx $CLIENT1_N_QUEUES $CLIENT2_N_QUEUES)
     ;;
  OVS)
     ovs-vsctl add-br ${TEST_BR_NAME}
     ;;
  esac

 brctl addbr ${CTRL_BR_NAME} 2>&1 | grep -v "already exists"
 ifconfig ${CTRL_BR_NAME} up
}
```

Reply:
Yes, but bridges are always set (it just depends what kind of bridge are set). In your case, we have a case were nothing will happen.